ilastik/utility/gui/roi.py

```diff
@@ -0,0 +1,66 @@
+###############################################################################
+#   ilastik: interactive learning and segmentation toolkit
+#
+#       Copyright (C) 2011-2014, the ilastik developers
```

Comment:
sorry to be so pedantic, but one could update the year here...

Reply:
Good catch.

However, we may not need to update these years at all [[1]] [[2]] [[3]]. As far as I understood, it's better to add a year each time you "modify" the code, that is, when you do either an initial release or copy a part of code from some other project, you add a date.

Finally, it would be great to have some sort of license checker as a part of our CI pipeline.

[1]: https://stackoverflow.com/a/2391555
[2]: https://opensource.stackexchange.com/a/1265
[3]: http://www.gnu.org/licenses/gpl-howto.html

Reply:
the date has been bugging me ever since, but since we have it and you added a new license... In any case we have two options:

Maybe we should discuss getting rid of the date, or even what our switch to MIT/BSD License is doing ;)